core/common/src/thrift/exception.thrift

```diff
@@ -1,5 +1,120 @@
 namespace java alluxio.thrift
 
+enum TExceptionStatus {
+  // Canceled indicates the operation was cancelled (typically by the caller).
+  CANCELED = 1,
```

Comment:
+1 for changing the data server to use these codes as well. I don't think the data server uses proto though, I think it would use our native `ExceptionStatus` enum.